tools/stress-cluster/services/otelcollector/otel-collector-config.yml

```diff
@@ -0,0 +1,33 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+      grpc:
+ 
+processors:
+  batch:
+ 
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [logging, azuremonitor]
+    metrics:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [prometheus, azuremonitor]
```

Comment:
The prometheus exporter is not configured, and should probably not be used for metrics.

Reply:
Removed!